src/fixed-footer-layout.tsx

```diff
@@ -135,28 +102,31 @@ const FixedFooterLayout: React.FC<Props> = ({
         };
     }, [hasContentEnoughVSpace, platformOverrides]);
 
-    const hasContentScroll = () => hasScroll(getScrollableParentElement(containerRef.current));
-
-    const classes = useStyles({
-        footerBgColor,
-        containerBgColor,
-        footerHeight: realFooterHeight,
-        windowHeight,
-        screenHeight,
-        isContentWithScroll: hasContentScroll(),
-        hasContentEnoughVSpace,
-    });
+    const isContentWithScroll = hasScroll(getScrollableParentElement(containerRef.current));
+    const isFixedFooter = hasContentEnoughVSpace || !isContentWithScroll;
 
     return (
         <>
-            <div ref={containerRef} className={classnames({[classes.containerSmall]: isTabletOrSmaller})}>
+            <div
+                ref={containerRef}
+                className={styles.container}
+                style={assignInlineVars({
+                    [styles.vars.backgroundColor]: containerBgColor ?? '',
+                    [styles.vars.footerHeight]: isFixedFooter
+                        ? typeof footerHeight === 'string'
+                            ? footerHeight
```

Comment:
this wont work if footerHeight is defined as String this way `'200'`

Reply:
actually, this is wrong, I should be using realFooterHeight here

Reply:
fixed